pydantic/json_schema.py

```diff
@@ -368,12 +673,30 @@ def _common_set_schema(self, schema: core_schema.SetSchema | core_schema.FrozenS
         return json_schema
 
     def generator_schema(self, schema: core_schema.GeneratorSchema) -> JsonSchemaValue:
+        """
+        Returns a schema that matches a generator schema.
+
+        Args:
+            schema (core_schema.GeneratorSchema): The schema.
+
+        Returns:
+            JsonSchemaValue: The generated JSON schema.
+        """
```

Comment:
```suggestion
        """
        # TODO: make sure that GeneratorSchema accepts values from JSON, or this JSON schema should be changed..
```
If you don't mind, I'd appreciate if you add a TODO comment like this (might need to shorten to fit on the line), as it's something we really should check before the official release.